terminal/TerminalServer.cpp

```diff
@@ -59,18 +63,13 @@ void runTerminal(shared_ptr<ServerClientConnection> serverClientState,
     // Data structures needed for select() and
     // non-blocking I/O.
     fd_set rfd;
-    fd_set wfd;
-    fd_set efd;
     timeval tv;
 
     FD_ZERO(&rfd);
-    FD_ZERO(&wfd);
-    FD_ZERO(&efd);
     FD_SET(masterfd, &rfd);
-    FD_SET(STDIN_FILENO, &rfd);
```

Comment:
Why delete line 70?  Without this, we will wait for a timeout when there is data to send to the client immediately.